src/api/app/views/webui/feeds/notifications.rss.builder

```diff
@@ -0,0 +1,20 @@
+xml.rss version: '2.0' do
+  xml.channel do
+    xml.title "#{@user.realname} (#{@user.login}) notifications"
+    xml.description "Event's notifications from #{@configuration['title']}"
+    xml.link url_for only_path: false, controller: 'main', action: 'index'
+    xml.language "en"
+    xml.pubDate Time.now
+    xml.generator @configuration['title']
+
+    @notifications.each do |notification|
+      xml.item do
+        xml.title notification.title
+        xml.description notification.description
```

Comment:
@hennevogel this is how I'm doing it for the daily emails:

https://github.com/evanrolfe/open-build-service/blob/320c4899c313e3fc6ab9ddb20f21ed78ccaccadc/src/api/app/views/daily_email_mailer/notifications.text.erb

https://github.com/evanrolfe/open-build-service/blob/320c4899c313e3fc6ab9ddb20f21ed78ccaccadc/src/api/app/models/notifications/base.rb#L15-L21